compiler/rustc_ast_passes/src/ast_validation.rs

```diff
@@ -889,48 +889,45 @@ fn validate_generic_param_order(
 ) {
     let mut max_param: Option<ParamKindOrd> = None;
     let mut out_of_order = FxHashMap::default();
-    let mut param_idents = vec![];
```

Comment:
We can even skip calling `FxHashMap::default()` in case `generics` is empty, but this cheap? It calls only `hashbrown::raw::sse2::Group::static_empty`

Reply:
Yeah, it's cheap.